documentation/source/getting_started/overview.rst

```diff
@@ -1,42 +1,24 @@
 .. _Getting-Started:
 
-Introducing OpenEBS
+OpenEBS Introduction
```

Comment:
Changing the main title will require change index.rst as well.

Reply:
Updated index.rst, through another pull request.